pallets/did/src/origin.rs

```diff
@@ -39,8 +39,8 @@ impl<OuterOrigin, DidIdentifier, AccountId> EnsureOrigin<OuterOrigin> for Ensure
 where
 	OuterOrigin: Into<Result<DidRawOrigin<DidIdentifier, AccountId>, OuterOrigin>>
 		+ From<DidRawOrigin<DidIdentifier, AccountId>>,
-	DidIdentifier: Default,
-	AccountId: Default,
+	DidIdentifier: From<AccountId>,
+	AccountId: From<AccountId32>,
```

Comment:
This is probably an incorrect solution. In any case, we should not require `Default` for `AccountId` types as it is not implemented anymore.